### ISSUE TITLE
Availability from the Homebrew

### ISSUE THREAD

**Issue by arixmkii** [VirtusLab/coursier-m1#7]
Would be a great addition if both aarch64 and x86_64 versions would be available for Homebrew via [courier hosted formula](https://github.com/coursier/homebrew-formulas/blob/master/coursier.rb) or [core formula](https://github.com/Homebrew/homebrew-core/blob/master/Formula/coursier.rb) (which is JVM only right now).

**Comment by alexarchambault** [comment]
@arixmkii Indeed. If anyone wants to update the formula for that, just feel free to send a PR for it.

**Comment by Gedochao** [comment]
aarch64 artifacts are now available in https://github.com/coursier/coursier releases.
Closing this.
